### PR TITLE
Add failing test from issue 272

### PR DESCRIPTION
**Pull request by Kijewski**
Reported by Ciprian Dorin Craciun ([@cipriancraciun]) in issue [#272].

[#272]: <https://github.com/rinja-rs/rinja/issues/272>
[@cipriancraciun]: <https://github.com/cipriancraciun>

### CODE REVIEW

File: testing/tests/include.rs [hunk 1]
```diff
@@ -53,3 +53,35 @@
 
     assert_eq!(template.render().unwrap(), "Hello, Alice!\nHowdy, Bob!");
 }
+
+// FIXME: <https://github.com/rinja-rs/rinja/issues/272>
```
[the remark this answers is not in the document]
Comment (Kijewski) [reply testing/tests/include.rs:57]: Jupp, that's actually the point.

**Review by GuillaumeGomez — APPROVED** [review]

File: testing/tests/include.rs [hunk 2]
```diff
@@ -53,3 +53,35 @@ fn test_include_macro() {
...
+
+    #[derive(Template)]
+    #[template(path = "block_in_include_base.html", block = "block_in_base")]
+    struct TmplBlockInBase;
+
+    #[derive(Template)]
+    #[template(path = "block_in_include_partial.html", block = "block_in_partial")]
+    struct TmplBlockInPartial;
+
+    assert_eq!(
```
[the remark this answers is not in the document]
Comment (Kijewski) [reply testing/tests/include.rs:75]: The function is more of a reminder, to have the "FIXME" as part of the code. In this regard, the `#[should_panic]` line is a bug that needs fixing. :)

[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply testing/tests/include.rs:75]: Makes the one who will work on the bug actually look what is the expected output. In the current case, we test the bug whereas I think we should test that we don't have the expected output (if it makes sense?).

[the remark this answers is not in the document]
Comment (Kijewski) [reply testing/tests/include.rs:75]: I am not 100% sure that I comprehend what you meant, but I think it's implicitly covered through the `expected` argument of `#[should_panic]`, which will fail on any change.

If I misunderstood you, maybe you could just push a commit to my PR.

[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply testing/tests/include.rs:75]: Arf, didn't see the `should_panic`. Sorry about that...

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**